backend/config/data_providers.yml

```
@@ -146,6 +149,12 @@ api_particulier:
         - pole_emploi_contact
         - pole_emploi_adresse
         - pole_emploi_inscription
+    pole_emploi_indemnites:
+      label: API Indemnités Pôle emploi
```

J'aurai gardé le nom "paiements" :

```suggestion
      label: API Paiements Pôle emploi
```